lib/ci-helper.ts

```diff
@@ -554,6 +559,17 @@ export class CIHelper {
                 await addComment(`Submitted as [${
                     coverMid}](https://public-inbox.org/git/${coverMid})`);
             } else if (command === "/allow") {
+                if (!argument) {
+                    argument = pr.author;
+                } else {
+                    try {
+                        await this.github.getGitHubUserName(argument);
+                    } catch (reason) {
+                        throw new Error(`User ${
+                            argument} is not a valid GitHub username.`);
+                    }
+                }
```

Comment:
Good idea! For simplicity, we could have this check even if we just obtained the account name via `await getPRAuthor()`.